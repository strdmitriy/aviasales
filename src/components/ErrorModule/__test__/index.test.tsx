import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import { ErrorModule } from '../'

const errors = {
    status: "500"
}

test('Render ErrorModule', () => {
    const renderer = render(<ErrorModule status={errors.status} />)
    expect(renderer).toMatchSnapshot()
})
