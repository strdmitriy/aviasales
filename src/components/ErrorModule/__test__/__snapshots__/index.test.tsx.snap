// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ErrorModule 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Layout__Column-sc-1i5hea1-1 ErrorModule__Wrapper-sc-10am3do-0 cAExk"
      >
        <p
          class="Description-rarlu0-0 hQAroG"
          font-size="24px"
        >
          Что-то пошло не так...
        </p>
        <p
          class="Description-rarlu0-0 eLOmnN"
          font-size="14px"
        >
          Статус: 500
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Layout__Column-sc-1i5hea1-1 ErrorModule__Wrapper-sc-10am3do-0 cAExk"
    >
      <p
        class="Description-rarlu0-0 hQAroG"
        font-size="24px"
      >
        Что-то пошло не так...
      </p>
      <p
        class="Description-rarlu0-0 eLOmnN"
        font-size="14px"
      >
        Статус: 500
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
